src/odemis/gui/comp/canvas.py

```diff
@@ -873,7 +873,11 @@ def set_images(self, im_args):
                 md['dc_shear'] = shear
                 md['dc_flip'] = flip
                 md['dc_keepalpha'] = keepalpha
-                md['blend_mode'] = blend_mode
+                # Stream._find_metadata discards 'blend_mode' from the metadata as it is considered not useful
+                # If 'blend_mode' is present in the metadata, it has been set by the user, in that case no need
+                # to update it with the value from DblMicroscopeCanvas._get_ordered_images logic
```

Comment:
That comment is confusing. You've just added code that makes find_metadata not discard blend_mode. Also the "user" here probably doesn't refer to the microscope/app user, but to the calling code, right? Please adjust this comment to make it clearer.